src/headless/main.cpp

```diff
@@ -72,9 +112,24 @@ static bool run_headless(Dataloader::path_vector_t const& roots, bool run_tests)
 		CountryInstanceManager const& country_instance_manager =
 			game_manager.get_instance_manager()->get_country_instance_manager();
 
-		print_ranking_list("Great Powers", country_instance_manager.get_great_powers());
+		std::vector<CountryInstance*> const& great_powers = country_instance_manager.get_great_powers();
+		print_ranking_list("Great Powers", great_powers);
 		print_ranking_list("Secondary Powers", country_instance_manager.get_secondary_powers());
 		print_ranking_list("All countries", country_instance_manager.get_total_ranking());
+
+		Logger::info("===== RGO test... =====");
+		for (int i = 0; i < std::min<int>(3, great_powers.size()); ++i) {
+			CountryInstance const& great_power = *(great_powers[i]);
+			ProvinceInstance const* const capital_province = great_power.get_capital();
+			if(capital_province == nullptr) {
+				Logger::warning(great_power.get_identifier(), " has no capital ProvinceInstance set.");
+			}
+			else {
+				print_rgo(*capital_province);
+			}
+		}
+		ProvinceInstance *const& new_orleans = game_manager.get_instance_manager()->get_map_instance().get_province_instance_by_identifier("139");
```

Comment:
```suggestion
		ProvinceInstance const* new_orleans = game_manager.get_instance_manager()->get_map_instance().get_province_instance_by_identifier("139");
```
Better to use a `const*`, and no need to make it a `const&` (the only reasons to take a reference to a pointer is if you want to change the value of a pointer variable somewhere else or have access to the value of a pointer somewhere else that might change, in all other cases it just adds an unnecessary extra level of indirection).
Also best to check if the `new_orleans` pointer is null before dereferencing it below, just so the program doesn't crash when tested with mods that don't have such a province (e.g. a test mod with the bare minimum define needed to run the game).

Reply:
This line will just go as it was a leftover from adhoc rgo testing